index.html

```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Review code</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <h1 id="title">Hello World ! , We are Software developers</h1>
+    <p id="description">
+      Thank you for taking the time to help us improve the platform
+    </p>
+
+    <form id="survey-form">
+      <div class="form-group">
+        <label id="name-label">Name</label>
+        <input type="text" id="name" placeholder="name here" />
+      </div>
+      <div class="form-group">
+        <label id="email-label">Email</label>
+        <input type="email" id="email" placeholder="email" required />
```

Comment:
add more details in placeholder so that user can easily understand.

Reply:
Awesome! kindly please rephrase the request in a more human-friendly way; not as an order I think. 👍